src/main/java/frc/utils/applicationsutils/OnDeploy.java

```diff
@@ -9,7 +9,13 @@ public static void main(String[] args) {
     }
 
     private static void startComputerPrograms() {
+        if (RobotConstants.ENABLE_KEYBOARD) {
+            runKeyboard();
+        }
+    }
 
+    private static void runKeyboard() {
+        CMDHandler.runPythonClass("keyboard/keyboard_to_nt_" + (RobotConstants.IS_RUNNING_ON_USB_B ? "usb_b" : "router") + ".py");
```

Comment:
```suggestion
        CMDHandler.runPythonClass("keyboard/keyboard_to_nt" + (RobotConstants.IS_RUNNING_ON_USB_B ? "usb_b" : "router") + ".py");
```

Reply:
Check if u can maybe insert like args In Java to python, so u don’t need 4 files per code

Reply:
what should i do about this thing, since you made it in another branch? should i merge that branch to me or just save it for another PR?

Reply:
> Check if u can maybe insert like args In Java to python, so u don’t need 4 files per code

what should i do about this thing, since you made it in another branch? should i merge that branch to me or just save it for another PR?